examples/config-builder/three_qb_system.py

```diff
@@ -0,0 +1,317 @@
+import numpy as np
+
+from qualang_tools.config.configuration import *
+from qualang_tools.config.components import *
+from qualang_tools.config.builder import ConfigBuilder
+
+cnot_coupler = ArbitraryWaveform(
+    "cnot_coupler",
+    (
+        2.0
+        * np.pi
+        * np.array(
+            [
+                -0.0022640274687561295,
+                0.17980755683999214,
+                0.390545132044947,
+                0.35038324027031814,
+                0.19550416635358,
+                0.024658258821028484,
+                -0.12822902284043466,
+                -0.1752966912118911,
+                -0.17921353908242568,
+                -0.15096289009237282,
+                -0.110322391784403,
+                -0.08696095347060069,
+                -0.0801578668706695,
+                -0.07016164311782273,
+                -0.053739705459874915,
+                -0.035772043817697045,
+            ]
+        )
+    ).tolist(),
```

Comment:
Also, why do we need tolist()? Can't the ArbitraryWaveform handle numpy?

Reply:
GUI config builder required all waveforms to be in serializable JSON format so I intentionally converted them to python lists, ArbitraryWaveform can handle numpy arrays